spark/src/test/scala/org/apache/spark/sql/delta/AutoCompactSuite.scala

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
```

Comment:
2024?

Reply:
aah... this is a global problem for all files. let fix this in a separate PR for all files.